packages/rainbow-sprinkles/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "rainbow-sprinkles",
-  "version": "0.14.2",
+  "version": "0.14.3",
```

We can leave this out, changesets handles versioning for us (which happens in a separate PR)

```suggestion
  "version": "0.14.2",
```